### PR TITLE
BucketSpaceFS using native ops

### PR DESCRIPTION
**Pull request by liranmauda**
### Explain the changes

BucketSpaceFS using native ops

Signed-off-by: liranmauda <liran.mauda@gmail.com>

### Testing Instructions:
1.  Use standalone and do NSFS Bucket operations

### CODE REVIEW

File: src/sdk/bucketspace_fs.js [hunk 1]
```diff
@@ -21,13 +41,13 @@ class BucketSpaceFS {
...
-            const entries = await fs.promises.readdir(this.fs_root, { withFileTypes: true });
-            const dirs_only = entries.filter(e => e.isDirectory());
+            const entries = await nb_native().fs.readdir(this.fs_config_param, this.fs_root);
+            const dirs_only = entries.filter(entree => isDirectory(entree));
             const buckets = dirs_only.map(e => ({ name: new SensitiveString(e.name) }));
             return { buckets };
         } catch (err) {
             if (err.code === 'ENOENT') {
-                console.error('nsfs: root dir not found', err);
+                console.error('BucketSpaceFS: root dir not found', err);
```
Comment (igorpick) [inline src/sdk/bucketspace_fs.js:50]: Don't we need to use the dbg?

Comment (liranmauda) [reply src/sdk/bucketspace_fs.js:50]: I don't think so. I think that we want this error.
@guymguym WDYT?

Comment (igorpick) [reply src/sdk/bucketspace_fs.js:50]: @liranmauda I mean instead of using 'console.error()', use dbg.error()

Comment (liranmauda) [reply src/sdk/bucketspace_fs.js:50]: Maybe, but in any case not in the scope of this PR. 
The dbg is not even required in this file.

**Review by igorpick — APPROVED** [review]

**Pull request merged**